packages/storybook/cxl-ui/cxl-playbook-accordion/theme=cxl-playbook.faq.stories.js

```diff
@@ -0,0 +1,57 @@
+import { html } from 'lit-html';
+import '@conversionxl/cxl-ui/src/components/cxl-app-layout.js';
+import { unsafeHTML } from 'lit-html/directives/unsafe-html';
+import faqData from '../cxl-vaadin-accordion/theme=cxl-faq.data.json';
+
+export default {
+  title: 'CXL UI/cxl-playbook-accordion',
+};
+
+const Template = () => html`
+  <style>
+    .plural .entry-title {
+      margin: 0;
+    }
+    cxl-playbook-accordion {
+      padding: 10px;
+    }
+  </style>
+
+  <cxl-app-layout theme="2c-l" id="container">
+    <cxl-playbook-accordion
+      id="cxl-vaadin-accordion-26107"
+      theme="reverse"
+      class="cxl-playbook-accordion-class"
+    >
+      ${faqData.map(
+        (el) => html`
+          <vaadin-accordion-panel
+            id="${el.cxl_hybrid_attr_post['@attributes'].id}"
+            class="${el.cxl_hybrid_attr_post['@attributes'].class}"
+            theme="reverse"
+          >
+            <div slot="summary">
+              <div class="left">
+                <vaadin-checkbox value="Option" theme="custom"></vaadin-checkbox>
+              </div>
+              <div class="right">
+                <div class="summary-top">${unsafeHTML(el.title.rendered)}</div>
+              </div>
+            </div>
+            <vaadin-vertical-layout>
+              <div class="item-content">
+                <div>${unsafeHTML(el.content.rendered)}</div>
+              </div>
+            </vaadin-vertical-layout>
+          </vaadin-accordion-panel>
+        `
+      )}
+    </cxl-playbook-accordion>
+  </cxl-app-layout>
+`;
+
+export const CxlAdeftVaadinAccordionThemeFaq = Template.bind({});
```

Comment:
First I've seen this `Template` object used in Storybook - what's it for?

Reply:
That was the new style of writing stories, which I get from Greg. 
The documentation gives an example here:
https://storybook.js.org/docs/web-components/writing-stories/args#story-args
I was sure this is the `brand new` style of writing stories.

Reply:
fixed in: https://github.com/conversionxl/aybolit/pull/87/commits/e3c683079e2f034fcea3a8c495c19987259661ac - is this now correct ?

Reply:
Good find. I think let's build all stories with existing model and refactor all at once in the future. Otherwise people will get confused on why stories are coded differently for no apparent reason.

Reply:
refactored